File: ConsoleApp1/ConsoleApp1/HopDongMuon.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QLTV
{
     class HopDongMuon
    {
		private DocGia dgNguoiMuon;
		private List<QuyenSachKhoaHoc> lDSSachKHMuon;
		private List<QuyenSachGiaoTrinh> lDSSachGTMuon;
		private int iSoLuongSachMuon;
		public DocGia NguoiMuon
		{
			get { return this.dgNguoiMuon; }
			set { this.dgNguoiMuon = value; }
		}
		public List<QuyenSachKhoaHoc> DSSachKHMuon
		{
			get { return this.lDSSachKHMuon; }
			set { this.lDSSachKHMuon = value; }
		}
		public List<QuyenSachGiaoTrinh> DSSachGTMuon
		{
			get { return this.lDSSachGTMuon; }
			set { this.lDSSachGTMuon = value; }
		}
		public int SoLuongSachMuon
		{
			get
			{
				return this.iSoLuongSachMuon;
			}
			set
			{
				this.iSoLuongSachMuon = value;
			}
		}

		public HopDongMuon()
		{
			this.lDSSachGTMuon = new List<QuyenSachGiaoTrinh>();
			this.lDSSachKHMuon = new List<QuyenSachKhoaHoc>();
		}
		public HopDongMuon(DocGia NguoiMuon, List<QuyenSachKhoaHoc> DSSachKHMuon, List<QuyenSachGiaoTrinh> DSSachGTMuon)
		{
			this.dgNguoiMuon = NguoiMuon;
			this.lDSSachKHMuon = DSSachKHMuon;
			this.lDSSachGTMuon = DSSachGTMuon;
			this.iSoLuongSachMuon = this.lDSSachGTMuon.Count+this.lDSSachKHMuon.Count;
		}		
	}
}

File: Quan_Ly_Thu_Vien/src/HopDongMuon.java
import java.util.ArrayList;

public class HopDongMuon
{
	private DocGia dgNguoiMuon;
	private ArrayList<QuyenSachKhoaHoc> lDSSachKHMuon;
	private ArrayList<QuyenSachGiaoTrinh> lDSSachGTMuon;
	private int iSoLuongSachMuon;
	
	public DocGia getdgNguoiMuon() {
		return dgNguoiMuon;
	}
	public void setdgNguoiMuon(DocGia dgNguoiMuon) {
		this.dgNguoiMuon = dgNguoiMuon;
	}
	public ArrayList<QuyenSachKhoaHoc> getlDSSachKHMuon() {
		return lDSSachKHMuon;
	}
	public void setlDSSachKHMuon(ArrayList<QuyenSachKhoaHoc> lDSSachKHMuon) {
		this.lDSSachKHMuon = lDSSachKHMuon;
	}
	public ArrayList<QuyenSachGiaoTrinh> getlDSSachGTMuon() {
		return lDSSachGTMuon;
	}
	public void setlDSSachGTMuon(ArrayList<QuyenSachGiaoTrinh> lDSSachGTMuon) {
		this.lDSSachGTMuon = lDSSachGTMuon;
	}
	public int getiSoLuongSachMuon() {
		return iSoLuongSachMuon;
	}
	public void setiSoLuongSachMuon(int iSoLuongSachMuon) {
		this.iSoLuongSachMuon = iSoLuongSachMuon;
	}
	
	public HopDongMuon()
	{
		this.lDSSachGTMuon = new ArrayList<QuyenSachGiaoTrinh>();
		this.lDSSachKHMuon = new ArrayList<QuyenSachKhoaHoc>();
	}
	public HopDongMuon(DocGia NguoiMuon, ArrayList<QuyenSachKhoaHoc> DSSachKHMuon, ArrayList<QuyenSachGiaoTrinh> DSSachGTMuon)
	{
		this.dgNguoiMuon = NguoiMuon;
		this.lDSSachKHMuon = DSSachKHMuon;
		this.lDSSachGTMuon = DSSachGTMuon;
		this.iSoLuongSachMuon = this.lDSSachGTMuon.size() +this.lDSSachKHMuon.size();
	}		
}
